recipes/pyroSAR/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyroSAR" %}
+{% set version = "0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1008a6ed23bc3509b511af634360bccfb187a81745cdc5a1760c8ad3aa47bbe4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - setuptools_scm
```

Comment:
Is this needed at runtime?

Reply:
Apparently not. I thought it was needed so that a user can get the package version at run time via `pyroSAR.__version__`. I have checked and it seems to work without it. Thanks for checking.